docs/_posts/2016-01-11-admin-manual.md

```diff
@@ -39,14 +39,16 @@ Development Machine(s):
 * Eclipse
 * A merge tool such as Meld
 
-# Creating a Fork
-Use the GitHub interface or a private instance to create a forked repository.
+<a id="latest-release">
+## Downloading the latest release
```

Comment:
This line is showing up as a part of the paragraph below it. Same with "Setting Up the Application Server," and "Naming Schemes." I think the link on the line above them is causing the issue. A completely blank line is probably needed above a heading